tensorflow/lite/tools/optimize/quantize_model.h

```diff
@@ -53,6 +56,7 @@ TfLiteStatus QuantizeModel(flatbuffers::FlatBufferBuilder* builder,
 TfLiteStatus QuantizeModel(flatbuffers::FlatBufferBuilder* builder,
                            ModelT* input_model, const TensorType& input_type,
                            const TensorType& output_type, bool allow_float,
+                           const TensorType& activations_type,
```

Comment:
This is the only call invoked by the python tflite converter code. So I think instead of adding activations_type to every QuantizeModel function here, perhaps we can create a new QuantizeModel function that just adds activations_type to this function signature. (I am concerned about callers of the c++ code getting unexpected breakages.)

Reply:
done